src/EventRegistry.h

```diff
@@ -68,8 +68,8 @@ class EventRegistry final {
     // themselves.
     void SetErrorHandler(std::string_view name);
 
-    string_list UnusedHandlers();
-    string_list UsedHandlers();
+    [[deprecated("Remove in v7.1 - Unused handlers are now found via UsageAnalyzer.")]] string_list UnusedHandlers();
+    [[deprecated("Remove in v7.1 - UsedHandlers() is not reliable.")]] string_list UsedHandlers();
```

Comment:
You changed the one usage of `UsedHandlers` to `AllHandlers`. Should the deprecation warning suggest the same?